BASE/src/Microsoft.ApplicationInsights/Metrics/Implementation/AutocollectedMetricsExtraction/RequestMetricsExtractor.cs

```diff
@@ -13,36 +14,99 @@
     /// </summary>
     internal class RequestMetricsExtractor : ISpecificAutocollectedMetricsExtractor
     {
-        private MetricSeries responseTimeSuccessSeries;
-        private MetricSeries responseTimeFailureSeries;
+        /// <summary>
+        /// The default value for the <see cref="MaxResponseCodeToDiscover"/> property.        
+        /// </summary>
+        public const int MaxResponseCodeToDiscoverDefault = 10;
```

Comment:
We do not do subcodes, right? I'd still bump it a bit as [per this list](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes), only legitimate/often 4xx and 5xx is already at the limit of 10.